CodeTuple-Solutions/medusa-plugin-user-roles
# [FIX] updating plugin setup for comptability with Medusa 1.26.x

### What does it do?

It adds the latest plugin setup to become compatible with Medusa v1.
It also adds a new distinguishable  Icon for the Settings entry.

### Why is it needed?

Right now, after installing the plugin and enabling it on the configuration, it simply is not shown in the settings, we cannot see the plugin anywhere.

### How to test it?
1. Follow the [Medusa guide](https://docs.medusajs.com/development/plugins/create#test-your-plugin) for creating your plugins.
2. Run the Medusa server.
3. Go to the settings and check the extension section.
4. Click on the `Roles & Permissions` section.

### Notes
